src/Application.php

```diff
@@ -58,6 +58,14 @@ public function __construct(ContainerBuilder $containerBuilder = null, array $va
                 ])
             );
         });
+
+        // Override the callback resolver with ours
+        $this['callback_resolver'] = $this->share(function () {
+            return new CallbackResolver(
+                $this,
+                new CallableResolver($this->containerInteropProxy)
```

Comment:
> should I register callable resolver in DI first?

yep sounds good. Maybe name it something like `phpdi.callable_resolver` to avoid crowding the root "namespace" for users?